example/two-pods-two-distinct-nx-gzip/05-deployment-first.yaml

```
@@ -31,16 +34,13 @@ spec:
         serviceAccountName: ex-sa
         securityContext:
           privileged: false
-          runAsUser: 1000
+          runAsUser: 1000740023
```

```suggestion
          runAsUser: 1000740023
```

This will not be stable

---

Let's create a service-account like https://github.com/IBM/power-device-plugin/blob/main/examples/01-sa.yaml

Add https://github.com/IBM/power-device-plugin/blob/main/examples/02-rbac.yaml *this one is for mounting-directories* 

Then add to the deployment https://github.com/IBM/power-device-plugin/blob/main/examples/03-deployment.yaml#L35

---

you have references to a service account in the example, so it might just need to be added to the container.